src/stories/examples/DirectoryStructure/DirectoryStructure.module.css

```diff
@@ -0,0 +1,97 @@
+.nodeWrapper {
+  align-items: center;
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  height: 32px;
+  padding-inline-end: 8px;
+  padding-inline-start: 8px;
+  border-radius: 4px;
+  cursor: pointer;
+  white-space: nowrap;
+  position: relative;
+  z-index: 3;
+}
+
+.nodeWrapper:hover {
+  background-color: rgba(0, 0, 0, 0.04);
+}
+
+.nodeWrapper .expandIconWrapper {
+  align-items: center;
+  font-size: 0;
+  cursor: pointer;
+  display: flex;
+  height: 24px;
+  justify-content: center;
+  width: 24px;
+  transform: rotate(0deg);
+}
+
+.nodeWrapper .expandIconWrapper.isOpen {
+  transform: rotate(180deg);
+  & svg path {
```

Comment:
Just realized I left this in. I don't use CSS modules at all. Is this supported with CSS modules?